### ISSUE TITLE
Learn how to use the classificationLearnerApp

### ISSUE THREAD

**Issue by elboyran** [DynaSlum/SateliteImaging#15]

**Comment by elboyran** [comment]
- Don't have a separate validation set from the training set. The validation set is creating during the application of the validation scheme.
- Prepare training data into table. Use 'VisualWord#' as column name.
- If it table format load first to the work-space and later from there, rather than from file.
- In the Learner App use all features as Predictors and the classes as Responses. 
- Chose the default 5-fold cross-validation scheme.
- Export in the work-space the full model or generate code.
